radio/src/datastructs_private.h

```diff
@@ -956,6 +956,7 @@ PACK(struct RadioData {
 
   // Radio level tabs control (global settings)
 #if defined(COLORLCD)
+  uint8_t modelSelectLayout:2;
```

Comment:
Any reason this shouldn't be NOBACKUP'd (and now that I look at it, most of the options afterwards)?